libs/sdk-ui-filters/api/sdk-ui-filters.api.md

```diff
@@ -426,6 +426,8 @@ export interface IRankingFilterDropdownProps {
     // (undocumented)
     customGranularitySelection?: ICustomGranularitySelection;
     // (undocumented)
+    enableRenamingMeasureToMetric?: boolean;
```

Comment:
Just a confirmation: should we add `enableRenamingMeasureToMetric?: boolean;` to `IRankingFilterProps` at https://github.com/gooddata/gooddata-ui-sdk/blob/e8018b771f93a9c0560dcb3d8c2e4a1771468728/libs/sdk-ui-filters/src/RankingFilter/RankingFilter.tsx#L12

and then pass the new prop down the line to `RankingFilterDropdown` component at https://github.com/gooddata/gooddata-ui-sdk/blob/e8018b771f93a9c0560dcb3d8c2e4a1771468728/libs/sdk-ui-filters/src/RankingFilter/RankingFilter.tsx#L65

As far as I've searched, it seems that `RankingFilter` component from the SDK is currently not used anywhere. However, adding an optional prop to it should not cause any harm. In addition, in the future if the component is used, the prop would be ready to be used.

Reply:
fixed